toolchain/check/cpp/import.cpp

```diff
@@ -824,6 +824,15 @@ static auto ImportClassObjectRepr(Context& context, SemIR::ClassId class_id,
 
   // TODO: Import vptr(s).
 
+  // The kind of base class we've picked so far. These are ordered in increasing
+  // preference order.
+  enum class BaseKind {
+    None,
+    Empty,
+    NonEmpty,
+    Polymorphic,
+  } base_kind = BaseKind::None;
```

Comment:
I usually expect this syntax to be used with non-named struct decls, not with named decls. Maybe split it?
```suggestion
  };
  BaseKind base_kind = BaseKind::None;
```